### DISCUSSION TITLE
GUI questions

### DISCUSSION THREAD

**Discussion by chkw-rks** [plugdata-team/plugdata#381] [General]
It is exciting to see this project evolve! However, whats the idea behind stretching the controls in the upper part of the GUI? Must it stretch with the window size or can the icons just sit in the top left corner with fixed spacing left and right?

<img width="1055" alt="Screenshot 2023-01-17 184315" src="https://user-images.githubusercontent.com/5701406/212974319-f6676b37-23f3-4f91-b05f-15f9ddca5226.png">

**Comment by timothyschoen** [comment]
<img width="1046" alt="Screenshot 2023-01-18 at 15 48 19" src="https://user-images.githubusercontent.com/44585538/213202380-24fec8d4-ccfc-4787-a219-76dec161fde5.png">


This will be fixed in the next version, by moving a bunch of stuff into a global app menu. Now it takes up so little space that we won't have to scale the buttons at all.
